commit 1eee0f3f5b32aa6812fff10ea6f6a3115c91e230 (refs/changes/83/3883/3)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-04-27T21:56:36+00:00 (1 year, 5 months ago)
    
    MORAY-467 Document that PutObject returns the new etag
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/docs/index.md b/docs/index.md
index a15f1eb..693b828 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -543,14 +543,18 @@ looks like:
         }
     }
 
+If the update has been successfully applied, then the new `etag` will be
+returned to the client.
+
 ### API
 
     var data = {
         email: 'mark.cavage@joyent.com',
         ismanager: false
     };
-    client.putObject('foo', 'mcavage', data, function (err) {
+    client.putObject('foo', 'mcavage', data, function (err, res) {
         assert.ifError(err);
+        console.log(res.etag);
     });
 
 ### Inputs
